components/cosmetic_filters/resources/data/element_picker.ts

```diff
@@ -968,4 +989,6 @@ if (!active) {
         btnQuitText)
       launchElementPicker(root)
     });
+} else {
+  active.setAttribute(pickerStatusAttribute, 'maximized')
```

Comment:
hey I don't think we need to add this attribute - we're already adding/removing a `minimized` CSS attribute and that's the only thing the MutationObserver above is doing.

Additionally, it doesn't seem like we ever remove this attribute, so I think it should be safe to remove.